app/models/defect.rb

```diff
@@ -172,7 +172,7 @@ def reference_number
     ReferenceNumber.new(sequence_number).to_s
   end
 
-  def set_completion_date(date = nil)
+  def set_target_completion_date(date = nil)
```

Comment:
should there be an equivalent method for the actual completion date too? Seems inconsistent to have one but not both.